Apps/Common/src/Messaging/DbOutboxStore.cs

```diff
@@ -69,43 +68,38 @@ public async Task StoreAsync(IEnumerable<MessageEnvelope> messages, Cancellation
                 AssemblyQualifiedName = m.Content.GetType().AssemblyQualifiedName!,
             },
         });
-        await this.dbContext.AddRangeAsync(outboxItems, ct);
-        await this.dbContext.SaveChangesAsync(ct);
+        await this.outboxDelegate.Queue(outboxItems, ct);
+        await this.outboxDelegate.Commit(ct);
 
         this.backgroundJobClient.Enqueue<DbOutboxStore>(store => store.DispatchOutboxItems(CancellationToken.None));
     }
 
     /// <summary>
-    /// Dispatches any outbox messages not sent.
+    /// Dispatches any outbox messages not sent
```

Comment:
```suggestion
    /// Dispatches any outbox messages not sent.
```
There is probably a warning regarding current documentation. Missing period at end of sentence.